zipline/assets/asset_writer.py

```diff
@@ -563,3 +560,9 @@ def _load_data(self, equities, futures, exchanges, root_symbols):
             exchanges=exchanges_output,
             root_symbols=root_symbols_output,
         )
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc_info):
+        pass  # Shouldn't dispose of the engine explicitly
```

Comment:
Ok, two things then:

1. If we want the asset writer to be a ctx manager, then we should update all consumers to dispose of it when they're done.  Right now, I think we've only updated the zipline data bundle.  However, given that it doesn't actually manage any resources right now that need explicit disposal, my tendency is to undo this and re-do it with all the call sites updated if/when there are resources to clean up.  @llllllllll What do you think?
1. We should also update this comment to explain why we shouldn't dispose of the engine.  For example, will it cause problems if we do, or is it just unnecessary?